Fix Nightly 29 Nov 2024

https://github.com/gardenlinux/gardenlinux/actions/runs/12080809256/job/33689108520
```
FAILED base/test/test_systemd.py::test_systemctl_no_failed_units - AssertionError: systemd units ssh.service failed
====== 1 failed, 67 passed, 108 skipped, 4 warnings in 303.49s (0:05:03) =======
```

--- a/features/ssh/file.include/etc/systemd/system-preset/00-sshsocket-disable.preset
+++ b/features/ssh/file.include/etc/systemd/system-preset/00-sshsocket-disable.preset
@@ -1,0 +1,1 @@
+disable ssh.socket

